server/kubernetes_api_workflow.go

```diff
@@ -128,6 +128,7 @@ cont:
 			return &t
 		}()
 	case pb.State_STATE_FAILED:
+		fallthrough
 	case pb.State_STATE_TIMEOUT:
```

Comment:
can we just do `case pb.State_STATE_FAILED, pb.State_STATE_TIMEOUT:` here? Doing it this way reinforces that they are both to have the same exact code.

Reply:
done